ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Must-Support-Flags.md

```diff
@@ -1,13 +1,34 @@
+Die Verwendung des MS-Flag an Profil-Elementen hat im Kontext dieses Leitfadens folgende Bedeutung:
 
 1. Im Kontext der Erstellung von FHIR-Ressourcen:
 
-    Wenn die entsprechende Information in einem der Testszenarien bekannt ist, dann MUSS das Feld in der erstellten Ressource gefüllt sein. Ist die Information nicht bekannt, KANN es leer bleiben.
+    Elemente, die mit MS gekennzeichnet sind, MÜSSEN vom erstellenden System implementiert werden. Dies bedeutet:
+    - Systeme MÜSSEN die entsprechenden Elemente in der eigenen Persistenz-Ebene vorhalten können. 
+    - Systeme MÜSSEN die entsprechenden Elemente in den entsprechenden Profil-Instanzen bei Abfrage über eine FHIR-Schnittstelle befüllen und ausgeben können.
+    - Systeme MÜSSEN die entsprechenden Elemente zur Anzeige bringen können (ausgenommen davon sind Systeme mit reiner Middleware-Funktion, die über keine grafische Nutzeroberfläche verfügen).
+    -  MS-Elemente, die als optional gekennzeichnet sind (0..-Kardinalität) KÖNNEN bei der Erstellung einer FHIR-Instanz fehlen, wenn die entsprechende Information im Einzelfall unbekannt, nicht zutreffend ist oder (noch) nicht erhoben wurde.
+    - Wenn MS-Elemente fehlen, SOLLEN Systeme über den Grund des Fehlens Informationen bereitstellen.
+
+    Eine **Ausnahme** stellen implizit vorhandene Informationen dar, die nicht aus der Persistenz-Ebene sondern nur aus dem Kontext des erstellenden Systems entnommen werden können. Beispiel: In einem System werden stets nur Informationen aktiver Patienten vorgehalten (inaktive Patienten werden verschoben/archiviert/gelöscht). Der Status eines Patienten wird daher nicht explizit angezeigt und in der Persistenz-Ebene gespeichert, sondern ergibt sich allein aus der Existenz des Datensatzes in diesem System. In solchen Fällen gilt:
+    - Bei lediglich implizit vorhandenen Informationen KANN der Inhalt eines MS-Elementes in der Schnittstelle hartkodiert werden (sofern gesichert ist, dass der entsprechende Wert des Elementes ausnahmslos auf alle Instanzen zutrifft, die dieses System erstellt).
+
+    Insbesondere für solche implizit vorhandene Informationen, können in den Profilen auf der Ebene einzelner, mit MS-Flag versehener Elemente konkretere Hinweise zur Implementierung enthalten sein, die die übergreifende Definition zu Must-Support für den Einzelfall konkretisieren, zum Beispiel zur Klarstellung wo und unter welchen Umständen hartkodierte Werte erlaubt sind.
+    MS-Flags, die nur unter bestimmten **Bedingungen** gelten, werden in Folge als 'Conditional Must Support' an entsprechender Stelle ausgewiesen (in der Spezifikation werden die MS-Definitions-Überschreibungen im Feld  'Definition' eines entsprechend mit MS-Flag versehenen Elements vorgehalten).
```

Comment:
Schwierig. wir erfinden hier zwei komplett neue Konzepte "Conditional Must Support" und "MS-Definitions-Überschreibung" ohne diese zu erläutern. Das klingt auch, als gäbe es ein separates "CMS"-Flag, was ja gar nicht der Fall ist.
Ich würde vorschlagen den ganzen Abschnitt zu vereinfachen:
"Ob für ein MS-Flag konkrete Ausnahmen gelten, in denen ein impliziter Wert hartcodiert gesetzt werden darf oder das MS-Flag nur unter bestimmten Bedingungen gilt, ist der Definition des jeweiligen Elementes zu entnehmen"

Reply:
Vorschlag übernommen